packages/desktop-lib/src/lib/strategies/concretes/remote-tracking-sleep.ts

```
@@ -0,0 +1,29 @@
+import { BrowserWindow } from 'electron';
+import { TrackingSleepStrategy } from '../abstracts/tracking-sleep.strategy';
+
+export class RemoteTrackingSleep extends TrackingSleepStrategy {
+	constructor(window: BrowserWindow) {
+		super(window);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)